services/voxtral-local/main.py

```
@@ -594,9 +591,135 @@ def _run_inference():
+            timeout=timeout_sec,
+        )
+        return transcription
+    except asyncio.TimeoutError:
+        logger.error(f"[REQ {req_id}] Generation timed out after {timeout_sec:.0f}s")
+        raise RuntimeError(f"Transcription timed out after {timeout_sec:.0f}s")
+
+
+async def _transcribe_streaming(
+    audio_array: NDArray[np.float32],
+    context: Optional[str],
+    language: Optional[str],
```

Ruff (0.14.13)

Unused function argument: `language`

(ARG001)